migrations/1668390172273-clientId-from.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class clientIdFrom1668390172273 implements MigrationInterface {
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE IF EXISTS public.pending_acks ADD COLUMN "from" character varying default '';
+      ALTER TABLE IF EXISTS public.pending_acks DROP CONSTRAINT IF EXISTS "PK_pending_acks";
+      ALTER TABLE IF EXISTS public.pending_acks ADD CONSTRAINT "PK_pending_acks" PRIMARY KEY ("messageId", "clientId", "from");
```

Comment:
3 column primary key here, just want to confirm it's intended.

Reply:
yes